javascript/javascript1/week3/smart-ease.js

```diff
@@ -0,0 +1,153 @@
+//Item array removal
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+
+const nameToRemove = "Ahmad";
+names.splice(1,1);  
```

Comment:
How do you know that you should remove the name on position 1? 
From the assignment: `Remove the item in names that is equal to nameToRemove` and that means that nameToRemove can change. 
Hint: `names.indexOf(nameToRemove)`